lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
@joneslee85 your solution is the same performance. Don't know which to use.
Found another trick
```ruby
      ENGINES = {
        'pry'  => :Pry,
        'ripl' => :Ripl'
        'irb'  => :IRB
      }.freeze
```
prevents from unneeded type casts inside ruby's `Object.const_defined?` and `Object.const_get`